apps/sms/test/unit/compose_test.js

```diff
@@ -684,19 +708,24 @@ suite('compose_test.js', function() {
       });
       test('Invokes the provided "failure" handler when the MozActivity fails',
         function(done) {
-        var req = Compose.requestAttachment();
-        req.onerror = function() {
-          assert.ok(true);
-          done();
-        };
+        var mockError = new Error();
+        Compose.requestAttachment().catch((err) => {
```

Comment:
nit: usually in ```catch``` tests I add ```.then(() => throw new Error('Success callback should not be called!'))``` so that test fails if we change code to not return rejected promise anymore.